packages/t/ttyplot/package.yml

```diff
@@ -1,16 +1,16 @@
 name       : ttyplot
-version    : 1.5.2
-release    : 1
+version    : 1.6.2
+release    : 2
 source     :
-    - https://github.com/tenox7/ttyplot/archive/refs/tags/1.5.2.tar.gz : a0279e55c1996133645437ccb02574c82d62f0baa9744065779b5667c1f1cb8d
+    - https://github.com/tenox7/ttyplot/archive/refs/tags/1.6.2.tar.gz : 99222721e2d89e1064546f29e678830ccaba9b75f276a4b6845cc091169787a0
 homepage   : https://github.com/tenox7/ttyplot
 license    : Apache-2.0
 component  : system.utils
 summary    : realtime plotting utility for terminal/console with data input from stdin
 description: |
     takes data from standard input / unix pipeline, most commonly some tool like ping, snmpget, netstat, ip link, ifconfig, sar, vmstat, etc. and plots in text mode on a terminal in real time
-setup      : |
-    %patch -p1 -i $pkgfiles/libtinfo.patch
+builddeps  :
+    - pkgconfig(ncursesw)
```

Comment:
It is not necessary to add `pkgconfig(ncursesw)` as build dependency because it is part of `system.devel`.